integration/nwo/fabric/network/checks.go

```diff
@@ -47,7 +47,11 @@ func (n *Network) CheckTopologyOrderers() {
 // CheckTopologyFSCNodes checks that the FSC nodes' are well configured.
 // It returns the parameters to be used for cryptogen.
 func (n *Network) CheckTopologyFSCNodes() (users map[string]int, userSpecs map[string][]topology.UserSpec) {
-	fscTopology := n.Context.TopologyByName("fsc").(*fsc.Topology)
+	t := n.Context.TopologyByName("fsc")
```

Comment:
I think it would be nice if you can define a topology without any FSC nodes - just for whatever testing reason.

I am actually wondering why the fabric (nwo) platform performs this check? Shouldn't every nwo platform be independent and than nwo will perform this check (from outside the platforms)?